libdnf/transaction/MergedTransaction.cpp

```diff
@@ -171,6 +172,21 @@ MergedTransaction::getConsoleOutput()
     return output;
 }
 
+
+static bool transaction_item_sort_function(const std::shared_ptr<TransactionItemBase> one, const std::shared_ptr<TransactionItemBase> two) {
+    if (one->isForwardAction() && one->isForwardAction()) {
```

Comment:
I believe that you forget to use two argument. I also suggest to rename one -> first, `two` to `second`.

Reply:
Good catch, thanks.
I also renamed the arguments to C++ish lhs/rhs.